e2eTests/src/test/java/no/elixir/e2eTests/GDIIntegrationTest.java

```diff
@@ -0,0 +1,7 @@
+package no.elixir.e2eTests;
+
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class GDIIntegrationTest {}
```

Comment:
@yasinmiran Could we have this class initially populated with all the same steps as above for FEGA?

Line 71 method name should probably be DownloadTest() instead of DownloadHackTest(). The actual testclass method called is correct, triggering the ExportRequest type of downloading, but the function name in the FEGAIntegrationTest class test pipeline fooled me for a short period ;) All: we are testing ExportRequest download in this implementation.